package.json

```diff
@@ -33,84 +33,87 @@
   ],
   "scripts": {
     "compile": "npx hardhat compile",
-    "deploy": "npx hardhat deploy",
```

Comment:
Updated dependencies so that everything worked. Now all our repos have the same versions